pages/common/clear.md

```diff
@@ -5,3 +5,15 @@
 - Clear the screen (equivalent to typing Control-L when using the bash shell):
 
 `clear`
+
+- Clear the screen but keep the terminal's scrollback buffer:
```

Comment:
I'd replace `terminal's scrollback buffer` with something simpler.

Reply:
What would you propose? Now I don't know something simpler to explain it, but I will think about it.

Reply:
Sounds way too technical, but understandable. If we don't come up with anything, this can stay, it's not a problem.

Reply:
Yes, I am agree with your argument. May be... "Clear the screen and allow to scroll up to the previously used commands" or "Clear the screen and allow to scroll up to the the previous typed commands"?

Reply:
I'd argue that we should keep it as-is. I'm not sure that it's any simpler if we change the language, and we'd risk being ambiguous if we were to change it.

Reply:
Nice! Thank you for your comment, @sbrl. I am going to mark it as resolved.